docs/conf.py

```diff
@@ -1,189 +1,133 @@
 # -*- coding: utf-8 -*-
 #
 # Configuration file for the Sphinx documentation builder.
-#
-# This file does only contain a selection of the most common options. For a
-# full list see the documentation:
-# http://www.sphinx-doc.org/en/master/config
-
 
-# If extensions (or modules to document with autodoc) are in another directory,
-# add these directories to sys.path here. If the directory is relative to the
-# documentation root, use os.path.abspath to make it absolute, like shown here.
-#
 import os
-import subprocess
 import sys
-import sphinx_rtd_theme
-from datetime import datetime
+import sciris as sc
 import fpsim as fp
 
 # Set environment
-os.environ['SPHINX_BUILD'] = 'True' # This is used so cv.options.set('jupyter') doesn't reset the Matplotlib renderer
+os.environ['SPHINX_BUILD'] = 'True' # This is used so fp.options.set('jupyter') doesn't reset the Matplotlib renderer
 os.environ['FPSIM_WARNINGS'] = 'error' # Don't let warnings pass in the tutorials
 on_rtd = os.environ.get('READTHEDOCS') == 'True'
 
-if sys.platform in ["linux", "darwin"]:
-    subprocess.check_output(["make", "generate-api"], cwd=os.path.dirname(os.path.abspath(__file__)))
-else:
-    subprocess.check_output(["make.bat", "generate-api"], cwd=os.path.dirname(os.path.abspath(__file__)))
 
-# Rename "fp_analysis" to "API reference"
-filename = 'modules.rst' # This must match the Makefile
-with open(filename) as f: # Read existing file
-    lines = f.readlines()
-lines[0] = "API reference\n" # Blast away the existing heading and replace with this
-lines[1] = "=============\n" # Ensure the heading is the right length
-with open(filename, "w") as f: # Write new file
-    f.writelines(lines)
+# -- Project information -----------------------------------------------------
 
+project = 'FPsim'
+copyright = f'2020 - {sc.now().year}, Bill & Melinda Gates Foundation. All rights reserved.\nThese docs were built for FPsim version {fp.__version__}.\n'
```

Comment:
Removing the period at the end of this line will get rid of the double period showing in the footer currently.